internal/gitstream/assign_test.go

```diff
@@ -215,6 +218,21 @@ reviewers:
 	})
 }
 
+func addEmptyCommit(t *testing.T, repo *git.Repository) (plumbing.Hash, *object.Commit) {
+	t.Helper()
+
+	wt, err := repo.Worktree()
+	require.NoError(t, err)
+
+	sha, err := wt.Commit("test message", &git.CommitOptions{AllowEmptyCommits: true})
```

Comment:
Didn't you want to add an author to that commit?

Reply:
In fact, it doesn't matter because we pass a commit object as parameter to `GetUser` - not the author's email.